ProcessLib/Parameter/CurveScaledParameter.h

```diff
@@ -19,19 +19,17 @@ namespace ProcessLib
 {
 template <typename T>
 struct CurveScaledParameter final : public Parameter<T> {
-    CurveScaledParameter(MathLib::PiecewiseLinearInterpolation const& curve,
+    CurveScaledParameter(std::string const& name_,
+                         MathLib::PiecewiseLinearInterpolation const& curve,
                          std::string const& parameter_name)
-        : _curve(curve),
-          _parameter_name(parameter_name)
+        : Parameter<T>(name_), _curve(curve), _parameter_name(parameter_name)
```

Comment:
I'm confused by Parameter name_ and parameter_name.
Renaming one of them would be IMO good idea.